changelog

```diff
@@ -1,3 +1,142 @@
+Version 0.10.9
+  * Big changes:
+    * Added an alien faction dwelling in a gas giant in the galactic Northeast. (@RisingLeaf, @GefullteTaubenbrust2)
+    * Added a faction of amphibious alien to the galactic south. (@Daeridanii1)
+    * Added Corroboration I, a new mission chain for the Gegno. (@Saugia)
+    * When attempting to capture a ship, an attacker's "attack" rating will always be pitted against the opposition "defense" rating, instead of only when the opposition chooses to defend. (@petervdmeer, @Hurleveur)
+    * Deprecated the use of "~" to denote half-additive image files, to avoid conflicts with Windows NT 8.3 file name compatibility. Half-additivity is now denoted with the "^" symbol. (@tibetiroka)
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@Quantumshark, @RisingLeaf, @stefanct, @Tangle10, @TheGiraffe3, @tibetiroka, @ziproot)
```

Comment:
```suggestion
      * Typo fixes. (@Hannah-E-M, @Quantumshark, @RisingLeaf, @stefanct, @Tangle10, @TheGiraffe3, @tibetiroka, @ziproot)
```

Reply:
Which PR was that? The one I found was in the last release.